lychee-lib/src/extract/html.rs

```diff
@@ -125,20 +125,3 @@ pub(crate) fn extract_html(buf: &str) -> Vec<RawUri> {
 
     tokenizer.sink.links
 }
-
-#[cfg(test)]
```

Comment:
i consolidated those tests in `extract/mod.rs` where the marginally-higher-level Extractor abstraction is tested

Reply:
thanks